language_service/src/hover/tests.rs

```diff
@@ -1596,3 +1596,16 @@ fn notebook_callable_defined_in_later_cell() {
         ("cell2", "operation Callee() : Unit {}"),
     ]);
 }
+
+#[test]
+fn notebook_local_definition() {
```

Comment:
What happens if the hover is on the reference to `x` in the second cell?

Reply:
The same thing that happens with hover on any local variable use with files: nothing. I'm not sure if that is intentional, but it is consistent at least. Is that worth filing an issue for follow up across both scenarios?